### ISSUE TITLE
[UI] Community Handbook Faq page need to be enhanced

### ISSUE THREAD

**Issue by ranjanmangla1** [layer5io/layer5#4518]
### Current Behavior

The faq page is very bland, there is no hierarchy to differentiate between questions and asnwers. And it does not look appealing like other faq pages on the site
![image](https://github.com/layer5io/layer5/assets/93506927/498c5178-1237-43d2-bde1-466ba8ddd277)

### Desired Behavior

It should be like this:
![image](https://github.com/layer5io/layer5/assets/93506927/9ac7ece1-25bc-4239-bf9c-e7fd68773781)
or we can convert into an Accordion :
![image](https://github.com/layer5io/layer5/assets/93506927/f21afc21-7c90-4815-afbc-b6fbfc5562ec)

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by ranjanmangla1** [comment]
please assign me this issue. i would love to work on it. Also, do tell between screenshot 1 or 2, which kind of faq you need

**Comment by leecalcote** [comment]
@ranjanmangla1 these FAQs show up on a number of pages. Each of those pages and both light and dark mode need to be considered in your proposal.

**Comment by ranjanmangla1** [comment]
@leecalcote no, i am not talking about all the faqs, i am just talking about the faq on https://layer5.io/community/handbook/faq, it does not look like all the faqs on other pages and also, hierarchy of text is not there, we can make it look like this: https://user-images.githubusercontent.com/93506927/252156626-9ac7ece1-25bc-4239-bf9c-e7fd68773781.png
or a faq accordion, which is used in the rest of website:
https://user-images.githubusercontent.com/93506927/252156732-f21afc21-7c90-4815-afbc-b6fbfc5562ec.png

**Comment by leecalcote** [comment]
I know that you’re not talking about all FAQs, which is why I am pointing out that what you’re suggesting, may affect the other pages in which all FAQs show up. Please attend to my asks or close this issue.

**Comment by ranjanmangla1** [comment]
how, all other pages have faqs in this form: ![accordion](https://user-images.githubusercontent.com/93506927/252156732-f21afc21-7c90-4815-afbc-b6fbfc5562ec.png), 
If you're talking about this form, we can convert it into a faq accordion like thisbecause as far as i have seen faqs on layer5.io, they are in this form, this will ensure design consistency and better way to show this faq. The current faq looks ugly in design.

**Comment by supzi-del** [comment]
Hey there! Can I work on this issue?

**Comment by leecalcote** [comment]
@ranjanmangla1, I misunderstood initially. I understand now and think that you have a great suggestion on your hands. Yes, this would be a nice improvement.

@ranjanmangla1, would you like to work on this or is this something that @supzi-del might tackle?

**Comment by ranjanmangla1** [comment]
Absolutely @leecalcote , just tell me which way you would like me to fix with accordion this way: 
![image](https://github.com/layer5io/layer5/assets/93506927/24cb015a-56a2-4f23-9726-465fa3ff3c66)
or this way: 
![image](https://github.com/layer5io/layer5/assets/93506927/fc4e802c-49e2-453a-9500-5a3852e48140)

**Comment by thisiskaransgit** [comment]
@ranjanmangla1, are you actively working on this or should I let @supzi-del tackle this?

**Comment by thisiskaransgit** [comment]
@supzi-del, please update this promptly

**Comment by Tabhi109** [comment]
If this issue is still open , I am willing to work this please assign it to me. @thisiskaransgit @leecalcote

**Comment by leecalcote** [comment]
With no response from @supzi-del, I assume that there is no active work being done here. Thanks for volunteering,, @Tabhi109

**Comment by Tabhi109** [comment]
@thisiskaransgit @leecalcote  , Can you please tell me which way do you want the FAQ page , so that I should start working on it.

**Comment by Tabhi109** [comment]
@thisiskaransgit , Okay thanks !

**Comment by Tabhi109** [comment]
@thisiskaransgit , issued a PR #4738 , Please review it , and do let me know if you need anything else.

**Comment by devarshishimpi** [comment]
Hey! I would love to contribute to this issue along with @gambhirsharma
We both would work on this in a team and provide you with the fix PR while keeping the Contributing Guidelines in mind.
Requesting the team to kindly assign the issue to both of us! Thanks!

**Comment by gambhirsharma** [comment]
Yes please assign us this issue.

**Comment by ankushbhardwaj408** [comment]
@Mohith234 If this issue is still open, could you please assign it to me? Thank you!

**Comment by Ashparshp** [comment]
Hi @leecalcote,

If this is still open, I'd like to take it on. Please consider assigning it to me, @thisiskaransgit .

**Comment by Ashparshp** [comment]
Hey @saurabh100ni, could you please confirm if this issue is still open, or if we can go ahead and close it?

**Comment by saurabh100ni** [comment]
@vishalvivekm //

**Comment by surafel58** [comment]
Is this issue still open ? @leecalcote

**Comment by leecalcote** [comment]
@surafel58 if you're truly interested, check and verify. Is it?

**Comment by userjaggu** [comment]
Hello @leecalcote , following up on your recent comment. I've checked the Community Handbook FAQ page, and I can confirm that the issue is still present—it is currently a plain list and not an accordion.

As this is a hacktoberfest issue and I'm eager to contribute, I would love to take this on. Please let me know if I can be assigned. Thank you!

**Comment by Rajesh-Nagarajan-11** [comment]
@userjaggu  Thank you for taking this, please go ahead 🚀

**Comment by bibhujit** [comment]
Hey @Rajesh-Nagarajan-11 / @leecalcote  - Looks like there is no matching PR for this issue. Can I pick this up? 

If I reading this rite,
convert this faq from src/sections/community/faq.js from its current state into better looking and review content if necessary. Someone in the thread advised to use accordion. I will like to give this a shot. Once this looks good, I could look at other faq pages. 

Thanks in advance.

**Comment by Rajesh-Nagarajan-11** [comment]
@userjaggu  any updates ? Are still working on this issue ?

**Comment by bibhujit** [comment]
if no one is working on this. please reassign to me.

**Comment by Rajesh-Nagarajan-11** [comment]
@bibhujit  All yours

**Comment by Alexjake660** [comment]
Is this issue still open, or has it been solved?
If it's still open, I'm interested in working on it.
